packages/hardhat-deploy-config/LICENSE

```diff
@@ -1,6 +1,6 @@
 (The MIT License)
 
-Copyright (c) 2023 Lightscale Inc.
+Copyright 2020-2023 Optimism
```

Comment:
We may need to also add MIT License of Lightscale as well as Optimism here, right?

Reply:
Then I wonder if we don't need to change the `author` of `packages/hardhat-deploy-config/package.json`.

Reply:
I also added to this package.